### DISCUSSION TITLE
New modes

### DISCUSSION THREAD

**Discussion by Hashimk2** [HasangerGames/suroi#396] [Polls]
Who would vote to add a mode that is similar to the  Potato event or the cobalt one or something similar to 50v50. The cobalt is you get to start off with a perk like making yourself small, cast iron skin, infinite ammo etc. It should also be like you get to choose which event you want to play in

**Comment by BringBack50v50** [comment]
Normal mode would get too boring the mode changes months after months

**Reply by ghost** [reply]
> Normal mode would get too boring the mode changes months after months

I agree

**Comment by EagleEye2010** [comment]
50v50 would be fun...

**Reply by Hashimk2** [reply]
Yeah, it would

**Comment by Crimson-Haze** [comment]
All of the above

**Reply by Crimson-Haze** [reply]
But if I had to choose, potato mode.

**Comment by kidwaih2** [comment]
potato event has been added(infection) and 50v50 is going to be added
